cognee/modules/data/methods/get_deletion_counts.py

```
@@ -0,0 +1,90 @@
+                    data_count_query = (
+                        select(func.count())
+                        .select_from(DatasetData)
+                        .where(DatasetData.dataset_id.in_(dataset_ids))
+                    )
+                    data_entry_count = (await session.execute(data_count_query)).scalar_one()
+                    counts["data_entries"] = data_entry_count
+                else:
+                    counts["data_entries"] = 0
+                return counts
+
+        raise CliCommandException(f"No User exists with ID {user_id}", error_code=1)
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)